configure

```diff
@@ -1048,7 +1052,9 @@ for ac_option in $@; do
 	--disable-libunity)       _libunity=no    ;;
 	--enable-opengl)          _opengl=yes     ;;
 	--disable-opengl)         _opengl=no      ;;
-	--enable-bink)            _bink=yes       ;;
+        --enable-dispmanx)        _dispmanx=yes   ;;
```

Comment:
Formatting in this and the following two lines is broken? Did you use tabs instead of spaces or something like that?